### PR TITLE
Adding Control Tower boilerplate file

### PR DESCRIPTION
**Pull request by annaone**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by andrew-glenn — DISMISSED** [review]

File: deploy_control_tower.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+== Deploy with Control Tower
+You can deploy {partner-product-short-name} in a customized AWS Control Tower environment to help you set up a secure, multi-account AWS environment using AWS best practices. For details, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/?did=sl_card&trk=sl_card[Customizations for AWS Control Tower].
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:2]:
```suggestion
You can deploy {partner-product-short-name} in a customized AWS Control Tower environment to help you set up a secure, multi-account AWS environment using AWS best practices. For details, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/[Customizations for AWS Control Tower^]. 
```

Comment (MarciaRieferJohnston) [reply deploy_control_tower.adoc:2]: Adding "^" to open this link in a new tab.

File: deploy_control_tower.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
+== Deploy with Control Tower
+You can deploy {partner-product-short-name} in a customized AWS Control Tower environment to help you set up a secure, multi-account AWS environment using AWS best practices. For details, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/?did=sl_card&trk=sl_card[Customizations for AWS Control Tower]. 
+
+The root directory of the {partner-product-short-name} Quick Start repo includes a *ct* folder with a manifest.yaml file to assist you with the AWS Cloud Tower deployment. This file has been customized for the {partner-product-name} Quick Start.
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:4]:
```suggestion
The root directory of the {partner-product-short-name} Quick Start repo includes a `ct` folder with a `manifest.yaml` file to assist you with the AWS Control Tower deployment. This file has been customized for the {partner-product-name} Quick Start. 
```

File: deploy_control_tower.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
+== Deploy with Control Tower
+You can deploy {partner-product-short-name} in a customized AWS Control Tower environment to help you set up a secure, multi-account AWS environment using AWS best practices. For details, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/?did=sl_card&trk=sl_card[Customizations for AWS Control Tower]. 
+
+The root directory of the {partner-product-short-name} Quick Start repo includes a *ct* folder with a manifest.yaml file to assist you with the AWS Cloud Tower deployment. This file has been customized for the {partner-product-name} Quick Start. 
+
+In the following sections, you will review and update the settings in this file and then upload it to the S3 bucket that is used for the deployment.
+
+=== Review the manifest.yaml file
+
+. Navigate to the root directory of the {partner-product-short-name} Quick Start, and open the manifest.yaml file, located in the *ct* folder.
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:10]:
```suggestion
. Navigate to the root directory of the {partner-product-short-name} Quick Start, and open the `manifest.yaml` file, located in the `ct` folder.
```

File: deploy_control_tower.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+. Confirm that the `region` attribute references the Region where AWS Control Tower is deployed. The default Region is us-east-1. You will update the `regions` attribute (located in the *resources* section) in a later step. 
+. Confirm that the `resource_file` attribute points to the public S3 bucket for the {partner-product-name} Quick Start. Using a public S3 bucket ensures a consistent code base across the different deployment options. For more information about the S3 bucket, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/?did=sl_card&trk=sl_card[Customizations for AWS Control Tower].
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:12]:
```suggestion
. Confirm that the `resource_file` attribute points to the public S3 bucket for the {partner-product-name} Quick Start. Using a public S3 bucket ensures a consistent code base across the different deployment options. For more information about the S3 bucket, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/[Customizations for AWS Control Tower^]. 
```

File: deploy_control_tower.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+=== Review the manifest.yaml file
+
+. Navigate to the root directory of the {partner-product-short-name} Quick Start, and open the manifest.yaml file, located in the *ct* folder.
+. Confirm that the `region` attribute references the Region where AWS Control Tower is deployed. The default Region is us-east-1. You will update the `regions` attribute (located in the *resources* section) in a later step. 
+. Confirm that the `resource_file` attribute points to the public S3 bucket for the {partner-product-name} Quick Start. Using a public S3 bucket ensures a consistent code base across the different deployment options. For more information about the S3 bucket, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/?did=sl_card&trk=sl_card[Customizations for AWS Control Tower]. 
+
+If you prefer to deploy from your own S3 bucket, update the path as needed.
+
+[start=4]
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:16]:
```suggestion
```

File: deploy_control_tower.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+In the following sections, you will review and update the settings in this file and then upload it to the S3 bucket that is used for the deployment.
+
+=== Review the manifest.yaml file
+
+. Navigate to the root directory of the {partner-product-short-name} Quick Start, and open the manifest.yaml file, located in the *ct* folder.
+. Confirm that the `region` attribute references the Region where AWS Control Tower is deployed. The default Region is us-east-1. You will update the `regions` attribute (located in the *resources* section) in a later step. 
+. Confirm that the `resource_file` attribute points to the public S3 bucket for the {partner-product-name} Quick Start. Using a public S3 bucket ensures a consistent code base across the different deployment options. For more information about the S3 bucket, see https://aws.amazon.com/solutions/implementations/customizations-for-aws-control-tower/?did=sl_card&trk=sl_card[Customizations for AWS Control Tower]. 
+
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:13]:
```suggestion
+
```

File: deploy_control_tower.adoc (hunk 5 above)
Comment (MarciaRieferJohnston) [reply deploy_control_tower.adoc:16]: Putting in a list continuation (the "+" in place of the blank line) in the previous step avoids the need to manually insert a start number.

File: deploy_control_tower.adoc (hunk 4 above)
Comment (MarciaRieferJohnston) [reply deploy_control_tower.adoc:12]: Adding "^" to open this link in a new tab.

File: deploy_control_tower.adoc (hunk 6 above)
Comment (MarciaRieferJohnston) [reply deploy_control_tower.adoc:13]: Adding a [list continuation](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/) (the "+" in place of the blank line) within block elements in an ordered-list item keeps autonumbering continuous.

File: deploy_control_tower.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+If you prefer to deploy from your own S3 bucket, update the path as needed.
+
+[start=4]
+. Review each of the `parameters` attributes and update them as needed to match the requirements of your deployment. 
+. Confirm that the `deployment_targets` attribute is configured for either your target accounts or organizational units (OUs). 
+. For the `regions` attribute, add the Region where you plan to deploy the {partner-product-short-name} Quick Start. The default Region is us-east-1.
+
+=== Upload the manifest.yaml file
+. Compress the manifest.yaml file and name it custom-control-tower-configuration.zip.
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:22]:
```suggestion
. Compress the `manifest.yaml` file and name it `custom-control-tower-configuration.zip`.
```

File: deploy_control_tower.adoc [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+. Upload the custom-control-tower-configuration.zip file to the S3 bucket that was created for the AWS Control Tower deployment (custom-control-tower-configuration-<accountnumber>-<region>).
```
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:23]:
```suggestion
. Upload the `custom-control-tower-configuration.zip` file to the S3 bucket that was created for the AWS Control Tower deployment (`custom-control-tower-configuration-<accountnumber>-<region>`).
```

**Review by MarciaRieferJohnston — CHANGES_REQUESTED** [review]
I added a few inline suggestions that can be merged directly.

File: deploy_control_tower.adoc (hunk 4 above)
Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc:12]: Is Control Tower the intended link and link text here?

Comment (MarciaRieferJohnston) [reply deploy_control_tower.adoc]: @annaone Hi, Anna. Just making sure that you saw my question here. Seems likely that this Control Tower URL and link text got pasted in where you probably meant to put an S3-related link?

Comment (annaone) [reply deploy_control_tower.adoc]: The link is correct, but we can just remove it, since we already have a pointer to that page earlier.

Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc]:
```suggestion
. Confirm that the `resource_file` attribute points to the public S3 bucket for the {partner-product-name} Quick Start. Using a public S3 bucket ensures a consistent code base across the different deployment options. 
```

Comment (MarciaRieferJohnston) [reply deploy_control_tower.adoc]: Done.

Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc]:
```suggestion
. Confirm that the `resource_file` attribute points to the public S3 bucket for the {partner-product-name} Quick Start. Using a public S3 bucket ensures a consistent code base across the different deployment options.
```

Comment (MarciaRieferJohnston) [inline deploy_control_tower.adoc]:
```suggestion
. Confirm that the `resource_file` attribute points to the public S3 bucket for the {partner-product-name} Quick Start. Using a public S3 bucket ensures a consistent code base across the different deployment options.
```

Comment (MarciaRieferJohnston) [reply deploy_control_tower.adoc]: @annaone Hi, Anna. I tried several times to delete that sentence and merge, but GitHub isn't letting me merge the suggested change. I get the message "Outdated suggestions cannot be applied." I'll see if I can find a way. If you get to it first, please go ahead. Ping me, and I'll approve it right away.

**Review by MarciaRieferJohnston — APPROVED** [review]

**Review by andrew-glenn — APPROVED** [review]

**Pull request merged**